build.sbt

```diff
@@ -32,7 +32,14 @@ lazy val compat = crossProject(JSPlatform, JVMPlatform)
   )
   .jvmSettings(
     OsgiKeys.exportPackage := Seq(s"scala.collection.compat.*;version=${version.value}"),
-    libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % "test"
+    libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % "test",
+    javaHome := {
+      val oldValue = javaHome.value
+      val isOnCi = sys.env.get("CI").isDefined
+
+      if (isOnCi) Some(file(sys.env("JAVA_HOME")))
+      else oldValue
```

Comment:
I'm not sure about this change.. it basically relies on a bug, that travis doesn't switch to jdk 6, but JAVA_HOME is set to 6 (https://github.com/travis-ci/travis-ci/issues/9713).

Reply:
@lrytz I force pushed this commit: 

https://github.com/scala/scala-collection-compat/pull/83/commits/3db34829572eadb25c42a8eb947cfcd2ff128625

Do you think this is ok?